metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -298,6 +304,29 @@ def make_tableau_client(self, site: str) -> Server:
             ) from e
 
 
+class GroupIngestionConfig(ConfigModel):
+    enable_workbooks: bool = Field(
+        default=True,
+        description="Whether or not to enable group ingestion for workbooks. "
+        "Default: True",
+    )
+
+    group_name_pattern: AllowDenyPattern = Field(
+        default=AllowDenyPattern.allow_all(),
+        description="Filter for Tableau group names when ingesting groups. "
+        "For example, you could filter for groups that include the term 'Consumer' in their name by adding '^.*Consumer$' to the allow list."
+        "By default, all groups will be ingested. "
+        "You can both allow and deny groups based on their name using their name, or a Regex pattern. "
+        "Deny patterns always take precedence over allow patterns. ",
+    )
+
+    displayed_capabilities: Optional[List[str]] = Field(
+        default=None,
+        description="List of Tableau permission capabilities (e.g. Read, Write, Delete) that will be displayed for each role in the custom property. "
```

Comment:
I'm kinda curious about the motivation behind this config option? My initial instinct was that we should just ingest all capabilities, without any filtering

Reply:
The initial motivation was that there's a lot of very specific capabilities in Tableau: https://help.tableau.com/current/server-linux/en-us/permissions_capabilities.htm. We thought this could be a bit confusing for users in the catalog that's why we added the possibility to only display certain capabilities in Datahub.

Since we're now moving most of the logic into a custom transformer now, I'm fine with removing this and showing all capabilities. Let me what you think.

Reply:
yeah let's do that